docs/configuring-socorro.rst

```diff
@@ -1,25 +1,68 @@
 .. index:: configuring-socorro
 
-Configuring Socorro
-===================
+Configure and run Socorro
+=========================
 
-Socorro produces graphs and reports, most are updated once per day.
+Initial setup
+-------------
 
-You must enter information about your releases into Socorro in order
-for this to work, and this information must match the incoming crashes.
+Socorro provides a setup script which attempts to initialize everything.
+If you have a distributed setup you should review this script and run
+the appropriate commands on the right servers::
+
+    sudo setup-socorro
+
+Start services
+--------------
+
+A full Socorro install requires the following to be running::
+
+    sudo systemctl enable socorro-collector
+    sudo systemctl enable socorro-processor
+    sudo systemctl enable socorro-middleware
+    sudo systemctl enable socorro-webapp
+
+Configure Nginx
+---------------
+
+Public-facing services like socorro-collector and socorro-webapp
+should be fronted by a webserver like Nginx.
 
-Configure Elasticsearch
------------------------
+For instance socorro-collector can be exposed via its uwsgi socket::
+    
+        location / {
+                uwsgi_pass unix:/var/run/uwsgi/socorro-collector.sock;
```

Comment:
Well the symbols upload problem is no more (we upload to S3 now), and everything runs as Socorro user, so I think we've nailed all of these.

However I'll make a note about creating this directory and making sure socorro user can write to it, since we know people who aren't using the RPM are trying to make these install instructions work (without looking at how the RPM does things)